src/dtls13.c

```diff
@@ -274,13 +274,7 @@ static int Dtls13GetRnMask(WOLFSSL* ssl, const byte* ciphertext, byte* mask,
         if (c->chacha == NULL)
             return BAD_STATE_E;
 
-        /* assuming CIPHER[0..3] should be interpreted as little endian 32-bits
```

Comment:
any reason to remove the comment? I think is useful

Reply:
No reason. Adding back in.